src/core/stream/representation/representation_stream.ts

```diff
@@ -377,13 +382,96 @@ export default function RepresentationStream<T>({
 
       case "parsed-segment":
         return initSegmentState.segmentData$
-          .pipe(mergeMap((initSegmentData) =>
-            pushMediaSegment({ clock$,
-                               content,
-                               initSegmentData,
-                               parsedSegment: evt.value,
-                               segment: evt.segment,
-                               segmentBuffer })));
+          .pipe(mergeMap((initSegmentData) => {
+            if (evt.value.inbandEvents !== undefined) {
+              const { manifestRefreshEvents,
+                      nonInterpretedEvents } = evt.value.inbandEvents
+                .reduce((acc, val: IInbandEvent) => {
+                  // Scheme that signals manifest update
+                  if (val.schemeId === "urn:mpeg:dash:event:2012" &&
```

Comment:
Weird to have DASH-specific things in the stream.
Couldn't that be already pre-parsed?